.circleci/config.yml

```diff
@@ -628,6 +628,9 @@ workflows:
       #  puppeteer-test, api-deploy-to-test or/and ui-deploy-to-test jobs.
       - wait_until_previous_workflow_done:
           <<: *filter-master-branch
+          requires:
```

Comment:
This seems a little indirect - another way to do this would be to have the `wait` be a set of reusable commands that you apply at the start of the commands you care about blocking.

Reply:
Yes, you're right about that. It's working well so far. 
But it feels right to me making the `wait_until_previous_workflow_done` job waits on `ui-unit-test` and `api-unit-test` jobs because `*-deploy-to-test` job also requires them.

I don't have a strong preference. Should I abandon this PR? because we have not seen any issue so far with it.

Or, turn the job into a reused command (will take a bit of more time). I can make this PR to do that if it is preferred.

Reply:
My preference would probably to either abandon or switch to command. I understand why you did it, but this job doesn't conceptually depend on those other jobs - we would just be abusing our knowledge of the other job's dependencies. It's also better to avoid introducing unnecessary restrictions on the job scheduler.